RedHatInsights/sed-frontend
# fix(Authentication): ESSNTL-5018 - Show "Not Authorised" with no permissions

# Description

Associated Jira ticket: #ESSNTL-5018 (issue)

This lifts up the RBAC check in sed-frontend in order to avoid querying the API with an account that doesn't have (any) permissions.


# How to test the PR

1) Create an account with no roles/permissions assigned (related to RHC)
2) Verify RHC shows the "Not authorised page"


# Checklist:
